### ISSUE TITLE
AlexNet.ipynb FC Layer Param number Wrong

### ISSUE THREAD

**Issue by SJLNNU** [LianHaiMiao/pytorch-lesson-zh#1]
hi,
When creating Class AlexNet, self.fc goes like this:
****************************************
self.fc = nn.Sequential(
            nn.Linear(1*1*256, 4096),
            nn.ReLU(inplace=True),
            nn.Dropout(0.5),
            nn.Linear(4096, 4096),
            nn.ReLU(inplace=True),
            nn.Dropout(0.5),
            nn.Linear(4096, n_class) # AlexNet上面是1000 ...如果测试的话用MNIST则可以使用10
        )
****************************************
but the first nn.Linear should be nn.Linear(6*6*256,4096).
When pytorch update to 1.0 some of the code in the project need to be changed like tensor.data[0] .
THX.

**Comment by SJLNNU** [comment]
Sorry  the first nn.Linear should be 1*1*256   ,becouse of the CIFAR10 resize of 96

**Comment by LianHaiMiao** [comment]
yeah, some of the code were out of date. I will take some time to update them. THX ~
